.travis.yml

```diff
@@ -0,0 +1,20 @@
+---
+language: ruby
+cache: bundler
+sudo: false
+bundler_args: --jobs 4 --retry 2 --without packaging documentation
+before_install:
+  - rm -f Gemfile.lock
+  - gem update --system && gem install bundler --no-document
+script:
+  - "bundle exec rake $CHECK"
```

Comment:
per comment below on rake tasks.  Consider using the pr::check task so there is a unified way to run all the checks...